File: Python/Easy/نوع القيمة المدخلة.py

# Creator: OmarAfet
# https://profile.satr.codes/OmarAfet/public/overview
# https://github.com/OmarAfet

def input_type(value: str) -> str:
    if value.isdigit():
        return 'integer'
    elif value.isalpha():
        return 'string'

    for i in value:
        if i == '.':
            return 'double'
        else:
            pass

File: JavaScript/Easy/نوع القيمة المدخلة.js
/*

Creator: OmarAfet
https://profile.satr.codes/OmarAfet/public/overview
https://github.com/OmarAfet

*/

function input_type(value) {
	if (/^[^\d]+$/.test(value)) {
		return "string";
	} else if (/^\d+$/.test(value)) {
		return "integer";
	} else if (/^\d+\.\d+$/.test(value)) {
		return "double";
	}
}
